views/wizard/user_add.tt

```diff
@@ -292,6 +292,53 @@
               }];
           -%]
         </div>
+        <div class="modal-frame" data-config='{"step":4,"frame":4,"item":null,"count":null}'>
```

Comment:
Whole new frame for delete reason - this isn't in the "list" on the header to avoid confusion when creating an account.